nexusadspy/client.py

```diff
@@ -125,7 +125,6 @@ def _do_throttled_request(self, url, method, params=None, data=None, headers=Non
                               sec_sleep=2., max_failures=100,
                               get_field=None):
 
-        data = json.dumps(data)
```

Comment:
I'd prefer to put this in a `try/catch` statement (`try: json.dumps(data) except [your exception]: data = data`), something like that